### PR TITLE
Ingested MT Summit 1997.

### PR DESCRIPTION
**Pull request by davidstap**
See https://aclanthology.org/www.mt-archive.info/90/MTS-1997-TOC.htm for original.

### CODE REVIEW

**Review by mjpost — COMMENTED** [review]
This is actually a scenario where the volume distinctions are clear enough that I'd one each for:

* tutorials
* plenaries (invited talks and panels)
* papers
* systems
* the workshop

I know in previous ones we've avoided over-splitting. My thinking here is that in those prior ones, it was a lot less clear how to divvy everything up. Here, it's clearer from the scheduled, and also the distinctions fall into clearly-defined volume conventions.

What do you think? Is this clear to you, too, or does it muddy the waters too much?

**Comment by davidstap** [comment]
It is clear to me too, but it is a bit hard to determine *exactly when* to split up. I'll change this one. Do you agree that all later editions could (and should?) also be split up?

**Comment by mjpost** [comment]
Let's try, but add some guidelines. 

1. Use volumes implied by page numbering, if available

Otherwise, group as follows. This usually implies grouping together papers, which may have been split up into smaller groups when creating the schedule. 

* Plenaries: invited talks, panels
* Papers: talks, posters
* System descriptions: if clearly distinguished from research talks and posters
* Workshops, one per, and themselves possibly subject to these rules

It isn't an error to do a single volume if the above distinctions are too difficult to make i'm a timely fashion.

File: data/xml/1997.mtsummit.xml [hunk 1]
```diff
@@ -0,0 +1,560 @@
...
+    </paper>
+    <paper id="51">
+      <title><fixed-case>MULTI</fixed-case>-lingual <fixed-case>DOC</fixed-case>umentation</title>
+      <author><first>Jörg</first><last>Schütz</last></author>
+      <pages>263-264</pages>
+      <url hash="fe23154c">1997.mtsummit-papers.51</url>
+    </paper>
+    <paper id="52">
+      <title><fixed-case>EST</fixed-case>eam <fixed-case>BTR</fixed-case></title>
+      <author><first></first><last>G.Magnusdottir</last></author>
```
Comment (danielgildea) [inline data/xml/1997.mtsummit.xml:363]: G. Magnusdottir

Comment (davidstap) [reply data/xml/1997.mtsummit.xml]: thanks for spotting, fixed it.

**Comment by mjpost** [comment]
@davidstap is this ready for re-review?

**Comment by davidstap** [comment]
It is; I have splitted the proceedings.

**Review by mjpost — APPROVED** [review]

**Pull request merged**